src/poliastro/cli.py

```diff
@@ -5,8 +5,7 @@
 import argparse
 
 import poliastro
-from poliastro import ephem
-
+from poliastro.neos.neos_dastcom5 import download_DASTCOM5
```

Comment:
I would move the module to `neos/dastcom5.py` so it reads `from poliastro.neos.dastcom5 import download_dastcom5` (yes, all lowercase)